testgrid/config.yaml

```diff
@@ -5809,6 +5813,12 @@ dashboards:
   - name: pr-test
     test_group_name: pull-cluster-api-provider-gcp-test
 
+- name: sig-gcp-compute-persistent-disk-csi-driver
```

Comment:
should we add this under the sig-gcp dashboard group?

Reply:
(up to you)

Reply:
it's going to be running sig storage tests, so maybe it could go under sig-storage?  or should we create a new dashboard group for csi drivers?

Reply:
I think the Kubernetes Integration tests in particular are more suited to either storage or drivers.
I would say it would go well in a group for drivers as when tests are failing in one driver, it's likely you will want to see if it is failing in other drivers too

Reply:
maybe group name "csi-drivers"

Reply:
alright so turns out after all that the CI basically forces me to go to `sig-gcp` which im not against so I'll just put it there.

Reply:
group name must be prefixed with sig. test name must be prefixed with group name. therefore `sig-gcp` kind of makes the most sense here